README.md

```diff
@@ -290,21 +297,152 @@ let decodedConversation = containerAgain.decode(with: client)
 try await decodedConversation.send(text: "hi")
 ```
 
-### Different types of content
+## Handle different content types
+
+All of the send functions support `SendOptions` as an optional parameter. The `contentType` option allows specifying different types of content other than the default simple string standard content type, which is identified with content type identifier `ContentTypeText`. 
+
+To learn more about content types, see [Content types with XMTP](https://xmtp.org/docs/dev-concepts/content-types).
```

Comment:
Added this link as I think it might be helpful for folx to understand the basic concept of content types.